full-service/src/db/transaction_log.rs

```diff
@@ -322,24 +327,38 @@ pub trait TransactionLogModel {
         conn: Conn
     ) -> Result<(), WalletDbError>;
 
-    /// Update the finalized block index to all pending transaction logs that associate with a given transaction output (txo).
+    /// Update the finalized block index to all pending transaction logs that have an output
+    /// transaction corresponding to `transaction_output_txo_id_hex`.
     /// 
     /// # Arguments
-    ///
-    ///| Name                    | Purpose                                                                      | Notes |
-    ///|-------------------------|------------------------------------------------------------------------------|-------|
-    ///| `txo_id_hex`            | The txo ID for which to get all transaction logs associated with this txo ID.|       |
-    ///| `finalized_block_index` | The block index at which the transaction will be completed and finalized.    |       |
-    ///| `conn`                  | An reference to the pool connection of wallet database                       |       |
-    ///
-    /// # Returns
-    /// * unit
+    /// * `transaction_output_txo_id_hex` - The txo ID for which to get all transaction logs
```

Comment:
I'm not sure the history on the doc format, if perhaps it was an early Cargo docs version or something, but the bullet + dash style is what's current
I mainly changed this as I was going to have to update the table format due to the argument rename, one of the reasons aligned table formats are avoided for these kind of docs.